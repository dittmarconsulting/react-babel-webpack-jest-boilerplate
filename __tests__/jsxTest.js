import React from 'react'
import {shallow} from 'enzyme'
import renderer from 'react-test-renderer'

import App from '../app/components/App'

test('App should render properly', () => {

  const component = renderer.create(
    <App/>
  )

  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
  //
  // // manually trigger the callback
  // tree.props.onMouseEnter()
  // // re-rendering
  // tree = component.toJSON()
  // expect(tree).toMatchSnapshot()
  //
  // // manually trigger the callback
  // tree.props.onMouseLeave()
  //
  // // re-rendering
  // tree = component.toJSON()
  // expect(tree).toMatchSnapshot()
});
