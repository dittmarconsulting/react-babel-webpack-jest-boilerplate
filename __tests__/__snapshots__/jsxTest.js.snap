exports[`test App should render properly 1`] = `
<h1>
  Hello World
</h1>
`;
